protocol/0044-LIME-lp_mechanics.md

```diff
@@ -336,11 +341,30 @@ In the case of spot markets it will be transferred into the network treasury for
+  - if `market.liquidity.providers.fee.calculationTimeStep` is set to `10s` and `validators.epoch.length` is set to `15s`, during the current epoch parameter change `market.liquidity.providers.fee.calculationTimeStep = 3s` is enacted, any funds that are in `ACCOUNT_TYPE_FEES_LIQUIDITY` account will be distributed to `ACCOUNT_TYPE_LP_LIQUIDITY_FEES` on the next block. For the next epoch the distribution will take place at `3` second (<a name="0044-LIME-062" href="#0044-LIME-062">0044-LIME-062</a>)
+- For a market that is in continuous trading if a new LP has active buy and sell orders on the market then makes a liquidity commitment to that market, at the start of the next epoch the active orders will count towards the LPs liquidity commitment. (<a name="0044-LIME-071" href="#0044-LIME-071">0044-LIME-071</a>)
+- If an LP with a liquidity provision and active orders on a market cancel their provision only (orders remain active), after the cancellation penalty (if it applies at end of epoch) the end of next epoch lp will not accrue any rewards or incur penalty for trades on the market. (<a name="0044-LIME-083" href="#0044-LIME-083">0044-LIME-083</a>)
+- For a market with market parameter `market.liquidity.priceRange = 0.05` (5%), is in continuous trading with `mid price` set to `5` and a new LP has committed liquidity:
+  - if during the epoch market parameter change `market.liquidity.priceRange = 0.1` (10%) is enacted and the LP has a buy order at price `4.74` and a sell order at price `5.25`, the LP is not meeting their committed volume of notional and a bond penalty will apply. The next epoch the parameter change takes effect, LP will meet their committed volume of notional and no bond penalty will apply. (<a name="0044-LIME-072" href="#0044-LIME-072">0044-LIME-072</a>)
+  - if the LP has a buy order at price `4.75` and a sell order at price `5.26`, the LP is not meeting their committed volume of notional (<a name="0044-LIME-073" href="#0044-LIME-073">0044-LIME-073</a>)
+  - if during the epoch market parameter change `market.liquidity.priceRange = 0.01` (1%)  the LP has a buy order at price `4.75` and a sell order at price `5.25`, the LP is meeting their committed volume of notional and no bond penalty will apply. The next epoch the parameter change takes effect, LP will not meet their volume of notional and bond penalty will apply. (<a name="0044-LIME-074" href="#0044-LIME-074">0044-LIME-074</a>)
+- For a market with market parameter `market.liquidity.priceRange = 0.5` (5%), is in monitoring auction with `last trade price` set to `5` and a new LP has committed liquidity:
+  - if the `indicative uncrossing price` is set to `4`, the LP has a buy order at price `3.79` and a sell order at price `5.25`, the LP is not meeting their committed volume of notional (<a name="0044-LIME-075" href="#0044-LIME-075">0044-LIME-075</a>)
```

Comment:
please make the ACs clear with what parameters apply, it's hard to scroll up to figure out what the range is. Also during auction we depend on last trade price as well as indicative uncrossing price.

Reply:
@davidsiska-vega does this read better?

`For a market with market parameter `market.liquidity.priceRange = 0.05` (5%), is in monitoring auction with `last trade price` set to `5`, `indicative uncrossing price` is set to `4` and the LP has a buy order at price `3.79` and a sell order at price `5.25`, the LP is not meeting their committed volume of notional because the price of the buy order is less than `5%` of `1-0.05 x min(5, 4) = 3.80``

Reply:
I made an update on the AC